server/web/errors/index.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+exports.register = function(server, options, next) {
+
+  server.ext("onPreResponse", function (request, reply) {
+
+    var response = request.response;
+    if (!response.isBoom) {
+        return reply.continue();
+    }
+
+    // Replace error with friendly HTML
+    var error = response;
+    var statusCode = error.output.statusCode;
+
+    switch(statusCode){
+      case 400:
+        return reply.view("errors/400").code(400);
+      case 403:
+        return reply.view("errors/403").code(403);
+      case 404:
+        return reply.view("errors/404").code(404);
+      default: 
+        return reply.view("errors/general");
```

Comment:
should `code` be set here to whatever `statusCode` is here? along that line, could determine view based on status and then return once

```javascript
var viewPath = "general";
switch(statusCode) {
  case 400:
    viewPath = statusCode;
  case 403:
    viewPath = statusCode;
  case 404:
    viewPath = statusCode
  // future viewPaths may not coincide directly w/ `statusCode`
}

return reply.view("errors/" + viewPath).code(statusCode);
```

food for thought